verifier/verifier.go

```diff
@@ -122,6 +129,15 @@ func NewVerifierWithOptions(ociTrustPolicy *trustpolicy.OCIDocument, blobTrustPo
 		}
 	}
 
+	revocationTimestampClient := verifierOptions.RevocationTimestampClient
+	if revocationTimestampClient == nil {
+		var err error
+		revocationTimestampClient, err = revocation.NewTimestamp(&http.Client{Timeout: 5 * time.Second})
```

Comment:
I guess the same question also applies to the code signing 2 seconds timeout at line 126? If we want to make it configurable, we could have a separate PR focusing on both code signing and timestamping revocation check.

Reply:
And according to Notary Project spec https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#crl-download
```
For each CDP location, [Notary Project verification workflow](https://github.com/notaryproject/specifications/blob/main/specs/signing-and-verification-workflow.md) will try to download the CRL for the default threshold of 5 seconds.
```
The number is indeed 5 seconds.

Reply:
@priteshbandi For timestamping revocation check, we will support CRL in the next iteration. But yes, we could keep the two seconds for now and have another PR to update it in the future.
>Also do we need separaterevocation.Revocation for timestampRevocation and codesiginngRevocation?

Yes, it is separated. There are two revocation clients under the `verifier` struct now, namely `revocationClient` and `revocationTimestampClient`.

Reply:
Based on @priteshbandi comment, updated to 2 seconds for this iteration.

/cc: @shizhMSFT

Reply:
>> Also do we need separaterevocation.Revocation for timestampRevocation and codesiginngRevocation?

>Yes, it is separated. There are two revocation clients under the verifier struct now, namely revocationClient and revocationTimestampClient. Both clients are [Revocation](https://github.com/notaryproject/notation-core-go/blob/e18808c29883e808525eb0516597e7a25b34b3fd/revocation/revocation.go#L29), but with different [Purpose](https://github.com/notaryproject/notation-core-go/blob/e18808c29883e808525eb0516597e7a25b34b3fd/revocation/ocsp/ocsp.go#L41).

Why do we need two clients? Can we reuse one client for both revocation checks?

Reply:
@priteshbandi Within the revocation workflow, we have a step to validate the certificate chain based on Notation requirements. Since the requirements for code signing and timestamping cert chains are different, we need a way to tell the `purpose` of the cert chain. The initial design of the `Revocation` interface did not provide such option for caller. Therefore, we need to have two clients for each purpose respectively. It's kind of an implementation challenge rather than designing.

In addition, decoupling revocation checks provides more flexibility to the caller. They may choose to use OCSP for code signing and CRL with cache for timestamping revocation check.